2020-09-23 22:23:24,087 INFO: Microblog startup [in D:\DEV\Dev\AnyProjects\study_flask_microblog\app\__init__.py:32]
2020-09-23 22:28:02,980 INFO: Microblog startup [in D:\DEV\Dev\AnyProjects\study_flask_microblog\app\__init__.py:32]
2020-09-23 22:28:04,550 ERROR: Exception on /index [GET] [in d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "D:\DEV\Dev\AnyProjects\study_flask_microblog\app\routes.py", line 39, in index
    return render_template("index.html", title='Home Page', form=form,
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\templating.py", line 137, in render_template
    return _render(
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "D:\DEV\Dev\AnyProjects\study_flask_microblog\app\templates\index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "D:\DEV\Dev\AnyProjects\study_flask_microblog\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: bootstrap/base.html
2020-09-23 22:28:51,137 INFO: Microblog startup [in D:\DEV\Dev\AnyProjects\study_flask_microblog\app\__init__.py:32]
2020-09-23 22:28:52,529 ERROR: Exception on /index [GET] [in d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "D:\DEV\Dev\AnyProjects\study_flask_microblog\app\routes.py", line 39, in index
    return render_template("index.html", title='Home Page', form=form,
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\templating.py", line 137, in render_template
    return _render(
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "D:\DEV\Dev\AnyProjects\study_flask_microblog\app\templates\index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "D:\DEV\Dev\AnyProjects\study_flask_microblog\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: bootstrap/base.html
2020-09-23 22:30:26,108 ERROR: Exception on /index [GET] [in d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py:1891]
Traceback (most recent call last):
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\_compat.py", line 39, in reraise
    raise value
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask_login\utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "D:\DEV\Dev\AnyProjects\study_flask_microblog\app\routes.py", line 39, in index
    return render_template("index.html", title='Home Page', form=form,
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\templating.py", line 137, in render_template
    return _render(
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\templating.py", line 120, in _render
    rv = template.render(context)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\jinja2\environment.py", line 1090, in render
    self.environment.handle_exception()
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\jinja2\environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\jinja2\_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "D:\DEV\Dev\AnyProjects\study_flask_microblog\app\templates\index.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "D:\DEV\Dev\AnyProjects\study_flask_microblog\app\templates\base.html", line 1, in top-level template code
    {% extends 'bootstrap/base.html' %}
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\templating.py", line 60, in get_source
    return self._get_source_fast(environment, template)
  File "d:\dev\dev\anyprojects\study_flask_microblog\venv\lib\site-packages\flask\templating.py", line 89, in _get_source_fast
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: bootstrap/base.html
2020-09-23 22:30:33,626 INFO: Microblog startup [in D:\DEV\Dev\AnyProjects\study_flask_microblog\app\__init__.py:32]
2020-09-23 22:31:41,067 INFO: Microblog startup [in D:\DEV\Dev\AnyProjects\study_flask_microblog\app\__init__.py:32]
2020-09-23 22:32:21,727 INFO: Microblog startup [in D:\DEV\Dev\AnyProjects\study_flask_microblog\app\__init__.py:32]
2020-09-23 22:33:49,366 INFO: Microblog startup [in D:\DEV\Dev\AnyProjects\study_flask_microblog\app\__init__.py:32]
2020-09-23 22:35:01,866 INFO: Microblog startup [in D:\DEV\Dev\AnyProjects\study_flask_microblog\app\__init__.py:33]
2020-09-23 22:35:53,309 INFO: Microblog startup [in D:\DEV\Dev\AnyProjects\study_flask_microblog\app\__init__.py:33]
2020-09-24 14:44:03,024 INFO: Microblog startup [in D:\DEV\Dev\AnyProjects\study_flask_microblog\app\__init__.py:35]
